routes/auth.js

```diff
@@ -15,8 +19,21 @@ router.post('/login', (req, res) => {
         res.json({ error: 'email ou mot de passe incorrect' })
       );
     }
-    const token = createToken(user);
-    res.json({ user, token });
+    connection.query('SELECT password FROM DVM_Legal_Entity WHERE email = ?', email, (error, results) => {
+      if (error) {
+        return (
+          res.status(500).json({ error })
+        );
+      } 
+      bcrypt.compare(password, results[0].password, function(err2, res2) {
```

Comment:
I guess you can find a better name than res2. look at https://www.npmjs.com/package/bcrypt